multy_core/src/bitcoin/bitcoin_transaction.cpp

```diff
@@ -716,8 +722,32 @@ void BitcoinTransaction::update()
             // not setting value with set_value(), since that would fail for read-only property.
             change_destination->amount.get_value() += remainder;
```

Comment:
it looks like rest of this change-adjusting code should be used only if change value is worth spending. Since in the `while`-loop we are reducing change size. Also, we should check if resulting change is worth spending.